benchmark/stargz_utils.go

```diff
@@ -135,8 +135,7 @@ func (proc *StargzContainerdProcess) StargzRpullImageFromECR(
 		containerd.WithSchema1Conversion,
 		containerd.WithPullUnpack,
 		containerd.WithPullSnapshotter("stargz"),
-		containerd.WithImageHandlerWrapper(source.AppendDefaultLabelsHandlerWrapper(
-			imageRef, "")),
+		containerd.WithImageHandlerWrapper(ctdsnapshotters.AppendInfoHandlerWrapper(imageRef)),
```

Comment:
not related to the change. but seems previously we use `source` from soci, not from stargz. Are they the same?

Reply:
Yeah its effectively the same since we  are not setting the prefetch size